### ISSUE TITLE
After SSO login using InAppBrowser plugin, it doesnot persist cookies while returning to app so unable to reload ios app

### ISSUE THREAD

**Issue by priyankabakkar** [apache/cordova-ios#1110]
# Bug Report

## Problem
We have hybrid app developed using cordova. Our app is “SwiftEnterprise” available on apple store.
We have functionality of SSO login which we are achieving using InAppBrowser plugin. Earlier it was working fine but from Aug 2020 it stopped working for ios.

We are able to login to SSO url, using InAppBrowser plugin but after login we were redirecting to our main app. Issue is, now we are not able to reload our app as jsession cookies are missing. Application not able to load. I guess it is cookies issue, I tried setting manual cookies as well but nothing works out.

### What is expected to happen?
After successful SSO login we redirect back to main application by closing inappbrowser. Here we expect to get jsession Id so that our session timeout will not happen and app can able to reload

### What does actually happen?
Currently on login we are getting session timeout because of missing cookies

### Environment, Platform, Device

Our app is hybrid app and we are facing this issue on IOS platform since aug 2020

**Comment by timbru31** [comment]
Thanks a lot for your issue, however the issue template exists for a reason. 😊   
It helps us to debug the issue further and to provide a solution much faster. With important information missing, we unfortunately can't help you.

Therefore, please edit this issue accordingly or close and create a new one and make sure to provide all the required information.

**Comment by priyankabakkar** [comment]
Yes I have added remaining details as well.. please go throught them

**Comment by oVerde** [comment]
Having the same issue, has anything being done in this case?
